discord_typings/resources/user.py

```diff
@@ -40,3 +41,24 @@ class UserData(UserBase):
 
 
 UserPremiumTypes = Literal[0, 1, 2]
+
+
+# https://discord.com/developers/docs/resources/user#connection-object
+
+
+@final
+class ConnectionData(TypedDict):
+    id: str
+    name: str
+    type: Literal["battlenet", "ebay", "epicgames", "facebook",
```

Comment:
Can you extract this into a separate alias like I originally recommended when you asked about creating this object? It could be called `ConnectionTypes`, and should also be exposed with `__all__`.

Reply:
Ahh my bad, I'll do that now